src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp

```diff
@@ -164,8 +165,8 @@ bool ov::intel_cpu::ACLInterpolateExecutorBuilder::isSupportedConfiguration(
     } else if (scale_h < 1 && scale_w < 1) {
         float down_scale_h = static_cast<float>(inp_shape[index_h]) / out_shape[index_h];
         float down_scale_w = static_cast<float>(inp_shape[index_w]) / out_shape[index_w];
-        bool int_factor =
-            down_scale_h == static_cast<int>(down_scale_h) && down_scale_w == static_cast<int>(down_scale_w);
+        bool int_factor = down_scale_h == static_cast<float>(static_cast<int>(down_scale_h)) &&
```

Comment:
double conversion

Reply:
We can avoid then in this case (since numbers are non-negative) using `std::floor`. Applied